test/api_endpoints/test_devices_endpoints.py

```
@@ -189,9 +198,10 @@ def test_devices_by_status(client, api_token, test_mac):
     assert fav_data is not None
     assert "&#9733" in fav_data["title"]
 
+
 def test_delete_test_devices(client, api_token, test_mac):
```

Ruff (0.14.5)

Unused function argument: `test_mac`

(ARG001)